api/src/test/java/com/epam/pipeline/controller/datastorage/DataStorageControllerTest.java

```diff
@@ -40,129 +33,30 @@
-    private static final String FROM_REGION = "fromRegion";
-    private static final String ID_PARAM = "id";
-    private static final String PATH = "path";
-    private static final String SHOW_VERSION = "showVersion";
-    private static final String PAGE_SIZE = "pageSize";
-    private static final String MARKER = "marker";
-    private static final String VERSION = "version";
-    private static final String TOTALLY = "totally";
-    private static final String TEST_PATH = "localhost:root/";
-    private static final String CONTENT_DISPOSITION_PARAM = "contentDisposition";
-    private static final String CONTENT_DISPOSITION_HEADER = "Content-Disposition";
-    private static final String CLOUD = "cloud";
-    private static final String SKIP_POLICY = "skipPolicy";
-    private static final String PIPELINE_ID = "pipelineId";
-    private static final String FILE_MASK = "fileMask";
-    private static final String FILTER_MASK = "filterMask";
-    private static final String REWRITE = "rewrite";
-    private static final String EMPTY_STRING = "";
-    private static final String PAGE = "page";
-    private static final String RUN_ID = "runId";
-    private static final String FILE_NAME = "file.txt";
-    private static final String FILE_SIZE = "0 Kb";
-    private static final String MULTIPART_CONTENT_TYPE =
-            "multipart/form-data; boundary=--------------------------boundary";
-    private static final String MULTIPART_CONTENT = "----------------------------boundary\r\n" +
-            "Content-Disposition: form-data; name=\"file\"; filename=\"file.txt\"\r\n" +
-            "Content-Type: application/octet-stream\r\n" +
-            "\r\n" +
-            TEST +
-            "\r\n" +
-            "----------------------------boundary--";
-    private static final ContentDisposition CONTENT_DISPOSITION = ContentDisposition.INLINE;
-    private static final Map<String, String> TAGS = Collections.singletonMap(TEST, TEST);
-    private final DataStorageRule dataStorageRule = DatastorageCreatorUtils.getDataStorageRule();
-    private final S3bucketDataStorage s3Bucket = DatastorageCreatorUtils.getS3bucketDataStorage();
-    private final DataStorageFile file = DatastorageCreatorUtils.getDataStorageFile();
-    private final DataStorageFolder folder = DatastorageCreatorUtils.getDataStorageFolder();
-    private final UpdateDataStorageItemVO update = DatastorageCreatorUtils.getUpdateDataStorageItemVO();
-    private final List<Pair<AbstractDataStorage, TypeReference>> storageTypeReferenceList =
-            DatastorageCreatorUtils.getRegularTypeStorages();
-    private final List<Pair<AbstractDataStorage, TypeReference>> securedStorageTypeReferenceList =
-            DatastorageCreatorUtils.getSecuredTypeStorages();
-
-    @Autowired
-    private DataStorageApiService mockStorageApiService;
+public class DataStorageControllerTest extends AbstractDataStorageControllerTest {
```

Comment:
Could you please move remaining data storage rules tests to the corresponding rules test class? Additionally restore, upload and tags tests should be moved to the items test class.